packages/driver/src/cypress/error_messages.ts

```diff
@@ -1803,6 +1834,18 @@ export default {
   },
 
   subject: {
+    is_null (obj) {
+      return stripIndent`\
+        ${cmd(obj.name)} failed because it requires a subject, but didn't receive one. This usually happens because you're trying to use a child query as if it were a parent.`
+    },
+    not_null (obj) {
```

Comment:
I assume this only occurs when a command cannot be chained? If so, using that verbiage

Reply:
As above - this is actually unused, and just some leftover code I forgot to clean up.